multiqc/utils/search_patterns.yaml

```diff
@@ -67,6 +67,8 @@ preseq:
     contents: 'EXPECTED_DISTINCT'
     # contents: 'TOTAL_READS	EXPECTED_DISTINCT'
     # contents: 'TOTAL_BASES	EXPECTED_DISTINCT'
+prokka:
+    contents: 'organism:'
```

Comment:
Yup, I think it will have to do. Only option for me is to add extra filtering options here I guess (eg. I thought that it supported regex matching until I checked, but that was elsewhere). But probably not worth it, the module should reject the file quickly enough. It's only really a speed / paranoia thing.